welcome/about-hcp.adoc

```diff
@@ -31,7 +31,7 @@ Use the following sections to find content to help you learn about and use {hcp-
 | Learn about {hcp-title} |Plan {hcp-title} deployment |Additional resources
 
 | xref:../architecture/index.adoc#architecture-overview[Architecture overview]
-| xref:../rosa_backing_up_and_restoring_applications/backing-up-applications.adoc#rosa-backing-up-applications[Back up and restore]
+| Back up and restore
```

Comment:
I am not quite sure what are you doing here, are you just trying to remove the link?

Reply:
Hi @wanghaoran1988  I am working on some build errors, so I haven't finished working in this area yet.